README.md

```diff
@@ -112,6 +123,11 @@ Validation errors occurred importing the spreadsheet.
 
 Then the indicated role, here `caseworker-cmc-loa1`, must be added to CCD (See [2. Add roles](#2-add-roles)).
 
+### NOTE: 
+The user and role creation can be done by executing the script
```

Comment:
Suggested edit: Initial user and role creation can be done ...

Reply:
Updated the readme.md file.